tensorflow/examples/udacity/1_notmnist.ipynb

```diff
@@ -496,22 +502,61 @@
         "outputId": "8af66da6-902d-4719-bedc-7c9fb7ae7948"
       },
       "source": [
+        "def make_arrays(nb_rows, img_size):\n",
+        "    if nb_rows:\n",
+        "        dataset = np.ndarray((nb_rows, img_size, img_size), dtype=np.float32)\n",
+        "        labels = np.ndarray(nb_rows, dtype=np.int32)\n",
+        "    else:\n",
+        "        dataset, labels = None, None\n",
+        "    return dataset, labels\n",
+        "\n",
+        "def merge_datasets(pickle_files, train_size, valid_size=0):\n",
```

Comment:
This and the test in `make_arrays` is so that the same function can be called for the training data and the test data. I can:

 1. Keep it like this
 2. Remove the default and make it explicit in the call for the test set
 3. Move the size test from `make_array` to `merge_datasets`
 4. Have a different function for the training+validation set and the test set
 5. Something else?